fastgen/utils/basic_utils.py

```
@@ -87,44 +87,65 @@ def to_str(obj: Any) -> str | Dict[Any, str]:
-        # 3. Enter strict inference mode (disables gradients, etc.) and autocast if needed
-        with torch.inference_mode(), torch.autocast(
-            dtype=precision_amp, device_type=device_type, enabled=precision_amp is not None
-        ):
-            yield
+            mod.train(mode)
+        yield
 
     finally:
-        # 4. Restore the original state of each module
+        # Restore the original state of each module
         for mod, was_training in zip(modules, previous_states):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)